project/AppDependencies.scala

```diff
@@ -3,19 +3,19 @@ import sbt.*
 object AppDependencies {
 
   private lazy val bootstrapPlayVersion = "7.19.0"
-  private lazy val hmrcMongoVersion     = "1.3.0"
+  private lazy val hmrcMongoVersion     = "1.6.0"
```

Comment:
Still update to bootstrap 7.23.0 for now.
mongo introduced changes in 1.4.0 for play 3. Revert until play 3 build. See https://github.com/hmrc/hmrc-mongo/releases